lockbox-ios/View/SettingListView.swift

```diff
@@ -59,17 +59,30 @@ extension SettingListView {
         self.dataSource = RxTableViewSectionedReloadDataSource(
                 configureCell: { _, _, _, cellConfiguration in
 
-                let cell = SettingCell(style: UITableViewCellStyle.value1, reuseIdentifier: "settings-cell")
+                let cell = UITableViewCell(
+                    style: cellConfiguration.cellStyle,
+                    reuseIdentifier: cellConfiguration.reuseIndicator)
+
                 cell.textLabel?.text = cellConfiguration.text
                 cell.selectionStyle = UITableViewCellSelectionStyle.none
 
+                if cellConfiguration.subtitle != nil {
+                    cell.detailTextLabel?.attributedText = cellConfiguration.subtitle
+                    cell.detailTextLabel?.numberOfLines = 0
+                } else if cellConfiguration.detailText != nil {
+                    cell.detailTextLabel?.text = cellConfiguration.detailText
+                }
+
                 if cellConfiguration.routeAction != nil {
                     cell.accessoryType = .disclosureIndicator
-                    cell.detailTextLabel?.text = cellConfiguration.detailText
                 } else if let switchSetting = cellConfiguration as? SwitchSettingCellConfiguration {
                     let switchItem = UISwitch()
                     switchItem.onTintColor = Constant.color.lockBoxBlue
-                    switchItem.addTarget(self, action: #selector(self.switchChanged), for: .valueChanged)
+                    if let onChanged = switchSetting.onChanged {
+                        switchItem.rx.value.changed.asObservable()
+                            .bind(to: onChanged)
+                            .disposed(by: self.disposeBag)
```

Comment:
This will cause some really subtle bugs with `RxSwift`. The cell may get swapped around / used for different cells when the tableview reloads / changes, but the binding will survive cell reuse and may get invoked but with the wrong observer. The correct way to work around this bug is to dispose of the stream via a `disposeBag` on the cell itself, and clear the DisposeBag every time the cell is being prepared for reuse (see `ItemDetailCell.swift` for an example of this behavior).